ARTICLE-1.markdown

```diff
@@ -80,13 +81,13 @@ In case the candidate is no longer in a position to become a
 Member-Owner at any stage of the process, either due to
 his/her own volition or due to the decision of the Member
 Body, CoLab shall refund either the $5000 USD capital buy-in,
-the $5000 USD via pro-bono work, or the full amount candidate
-would have charged CoLab for the 100 hours of work done as
-pro bono towards membership buy-in in case the “$50 USD per
-hour” option was taken.
+the $5000 USD via hours worked toward capital investment, or
```

Comment:
@brodavi , you said in the email that you are adding the wording saying CoLab will not refund. Here it still says we will refund